<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuildConfigurationsPage</class>
 <widget class="QWidget" name="BuildConfigurationsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="buildConfigurationsCompilerOptionsGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Compiler options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_13">
      <item>
       <widget class="QCheckBox" name="buildConfigurationsPreprocessOnly">
        <property name="text">
         <string>&amp;Preprocess only</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="buildConfigurationsCompile">
        <property name="text">
         <string>Preprocess and &amp;compile</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="buildConfigurationsMakeExecuteFile">
        <property name="text">
         <string>&amp;Make execute file</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="buildConfigurationsCustomCompilerOption">
          <property name="text">
           <string>C&amp;ustom:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="buildConfigurationsCompilerOption"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="buildConfigurationsExecuteOptionsGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Execute options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_15">
      <item>
       <widget class="QCheckBox" name="buildConfigurationsNoExecute">
        <property name="text">
         <string>&amp;No execute</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="buildConfigurationsWithDebug">
        <property name="text">
         <string>With &amp;debug</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_8">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>113</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buildConfigurationsPreprocessOnly</tabstop>
  <tabstop>buildConfigurationsCompile</tabstop>
  <tabstop>buildConfigurationsMakeExecuteFile</tabstop>
  <tabstop>buildConfigurationsCustomCompilerOption</tabstop>
  <tabstop>buildConfigurationsCompilerOption</tabstop>
  <tabstop>buildConfigurationsNoExecute</tabstop>
  <tabstop>buildConfigurationsWithDebug</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buildConfigurationsPreprocessOnly</sender>
   <signal>stateChanged(int)</signal>
   <receiver>BuildConfigurationsPage</receiver>
   <slot>onStateChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>18</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buildConfigurationsCompile</sender>
   <signal>stateChanged(int)</signal>
   <receiver>BuildConfigurationsPage</receiver>
   <slot>onStateChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>24</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>59</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buildConfigurationsMakeExecuteFile</sender>
   <signal>stateChanged(int)</signal>
   <receiver>BuildConfigurationsPage</receiver>
   <slot>onStateChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>82</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buildConfigurationsCustomCompilerOption</sender>
   <signal>stateChanged(int)</signal>
   <receiver>BuildConfigurationsPage</receiver>
   <slot>onStateChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>19</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buildConfigurationsNoExecute</sender>
   <signal>stateChanged(int)</signal>
   <receiver>BuildConfigurationsPage</receiver>
   <slot>onStateChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>14</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buildConfigurationsWithDebug</sender>
   <signal>stateChanged(int)</signal>
   <receiver>BuildConfigurationsPage</receiver>
   <slot>onStateChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>16</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buildConfigurationsCompilerOption</sender>
   <signal>editingFinished()</signal>
   <receiver>BuildConfigurationsPage</receiver>
   <slot>onTextChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onStateChanged(int)</slot>
  <slot>onTextChanged()</slot>
 </slots>
</ui>
